container/features/src/main/resources/features-minion.xml

```diff
@@ -28,4 +28,10 @@
       <feature>minion-core-api</feature>
       <bundle>blueprint:mvn:org.opennms.features/org.opennms.features.discovery/${project.version}/xml/blueprint-discovery-distPollerDaoMinion</bundle>
     </feature>
+    
+    <feature name="opennms-poller-monitors-core" description="OpenNMS :: Poller :: Monitors :: Core" version="${project.version}">
```

Comment:
The name of this feature cannot be the same as the other poller-monitors-core feature. Please rename it to "opennms-poller-monitors-core-minion".